src/webtoon_downloader.py

```diff
@@ -432,7 +454,8 @@ def main():
             return
     series_url = args.url
     separate = args.seperate or args.separate
-    download_webtoon(series_url, args.start, args.end, args.dest, args.images_format, args.latest, separate)
+    compress_cbz = args.cbz and args.separate
```

Comment:
This is always false when using the depracted --seperate option
```suggestion
    compress_cbz = args.cbz and separate
```

Reply:
Doh, yeah. Good spot.